app/components/CrashDiagramCard.tsx

```diff
@@ -148,7 +155,7 @@ export default function CrashDiagramCard({ crash }: { crash: Crash }) {
               setRotation={setRotation}
               zoomToImage={zoomToImage}
             />
-            <TransformComponent>
+            <TransformComponent contentStyle={{ mixBlendMode: "multiply" }}>
```

Comment:
this causes the crash diagram background white color to mix with our off-white background